### PR TITLE
add Python 3.13 to CI

### PR DESCRIPTION
**Pull request by belm0**

### CODE REVIEW

File: src/trio_util/_iterators.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from collections.abc import AsyncIterator
+from collections.abc import AsyncIterator  # pylint: disable=no-name-in-module
```
Comment (belm0) [inline src/trio_util/_iterators.py:1]: I don't know why this is necessary.

```
src/trio_util/_iterators.py:1:0: E0611: No name 'AsyncIterator' in module 'collections.abc' (no-name-in-module)
```

**Review by jrfk — APPROVED** [review]
arigato! :waiwai:

File: src/trio_util/_async_value.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # pylint: disable=line-too-long,multiple-statements
 from contextlib import contextmanager
-from typing import TypeVar, Generic, AsyncIterator, Tuple, ContextManager, Callable, overload
+from typing import TypeVar, Generic, AsyncIterator, Tuple, Callable, overload
+try:
+    from typing import ContextManager
+except ImportError:
+    from contextlib import AbstractContextManager as ContextManager  # pylint: disable=ungrouped-imports
```
Comment (belm0) [inline src/trio_util/_async_value.py:7]: this is the first user-visible change in a few years, so it needs a new package release

Comment (belm0) [reply src/trio_util/_async_value.py:7]: it's pushed to v0.8.0

I also invited you as pypi package owner

**Pull request merged**